patch/cisco-pmbus-driver-add-support-for-additional-temp-sensor.patch

```diff
@@ -0,0 +1,116 @@
+From aa09f80eb343110869b796205a45c16ed8951dea Mon Sep 17 00:00:00 2001
+From: Madhava Reddy Siddareddygari <msiddare@cisco.com>
+Date: Tue, 1 Jun 2021 19:31:36 -0700
+Subject: [PATCH] hwmon: (pmbus) Support fourth PSU temperature sensor
+
+PSU650W has 4 temperature sensors and pmbus driver has support
```

Comment:
how does this impact other PSU which only has 3 temperature sensors?

does this have negative impact to other PSU which only has less than 4 temperature sensors?

Reply:
Code changes will not have any impact on PSU with only 3 temperature sensors. Driver checks for the presence of each sensor and acts accordingly.

Please see the code snippet:
        if (pmbus_check_word_register(client, 0, PMBUS_READ_TEMPERATURE_1))
                info->func[0] |= PMBUS_HAVE_TEMP;
        if (pmbus_check_word_register(client, 0, PMBUS_READ_TEMPERATURE_2))
                info->func[0] |= PMBUS_HAVE_TEMP2;
        if (pmbus_check_word_register(client, 0, PMBUS_READ_TEMPERATURE_3))
                info->func[0] |= PMBUS_HAVE_TEMP3;